app/test_main.py

```diff
@@ -1,4 +1,27 @@
 from app.main import get_coin_combination
 
+import pytest
 
-# write your tests here
+
+@pytest.mark.parametrize(
+    "cents, expected_result",
+    [
+        pytest.param(
+            -5,
+            [0, 0, 0, 0],
+            id="should treat negative numbers as zero"
+        ),
+        pytest.param(
+            0,
+            [0, 0, 0, 0],
+            id="should return 4 zeros when 0 cents given"
+        ),
+        pytest.param(
```

Comment:
Omit using `pytest.param` with `id` here, there is no need to do it here. Pass the required arguments directly instead.